From 17fc6a2d2daae9e93957221c656b4b58418c0e12 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Thu, 2 Jun 2016 15:30:26 +0800
Subject: [PATCH 1079/5242] MLK-12876: mipi csi: Remove regulator enable code
 when driver probe

commit  b45174bd3c915096718504e9a9f5650490be04ea from
https://source.codeaurora.org/external/imx/linux-imx.git

Mipi CSI PHY regulator will enabled in function of s_power.
So remove regulator enable code when driver probe.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
(cherry picked from commit c29dda8f4092cdeff7f7661f8ac2a05f98a71296)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/media/platform/mxc/capture/mxc_mipi_csi.c |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/media/platform/mxc/capture/mxc_mipi_csi.c b/drivers/media/platform/mxc/capture/mxc_mipi_csi.c
index f3dd09c..483519b 100644
--- a/drivers/media/platform/mxc/capture/mxc_mipi_csi.c
+++ b/drivers/media/platform/mxc/capture/mxc_mipi_csi.c
@@ -380,11 +380,6 @@ static int mipi_csis_phy_init(struct csi_state *state)
 	ret = regulator_set_voltage(state->mipi_phy_regulator,
 			1000000, 1000000);
 
-	ret = regulator_enable(state->mipi_phy_regulator);
-	if (ret) {
-		dev_err(state->dev, "failed to set mipi regulator.\n");
-		return ret;
-	}
 	return ret;
 }
 
-- 
1.7.9.5

